### PR TITLE
DBTP-558 Get copilot-bootstrap image into version control

### PR DESCRIPTION
**Pull request by lgarvey**
Addresses https://uktrade.atlassian.net/browse/DBTP-558

### CODE REVIEW

**Review by WillGibson — COMMENTED** [review]
Probably worth adding a small README in here covering where it is published to etc. as it's not automated.

File: images/bootstrap/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM nginx:latest
+
+EXPOSE 8080
+
+COPY default.conf.template /etc/nginx/templates/default.conf.template
+COPY index.html /usr/share/nginx/html
+
```
[the remark this answers is not in the document]
Comment (WillGibson) [reply images/bootstrap/Dockerfile]: I'm not sure what you mean here @acodeninja, are you able to elaborate?

[the remark this answers is not in the document]
Comment (acodeninja) [reply images/bootstrap/Dockerfile]: For example, the boostrap image will not work with:

```yaml
image:
  location: xxxxxxxxxx.dkr.ecr.xxxxx.amazonaws.com/intranet/web:${IMAGE_TAG}
entrypoint:
  - web
```

[the remark this answers is not in the document]
Comment (WillGibson) [reply images/bootstrap/Dockerfile]: How would we add the other entry points? This feels like scope creep to me, I've created https://uktrade.atlassian.net/browse/DBTP-512 to define the scope of this pull request, but happy to create a second ticket for it if there is genuine need.

Create links from `/web`, `/celery` etc. to `/docker-entrypoint.sh`?

[the remark this answers is not in the document]
Comment (acodeninja) [reply images/bootstrap/Dockerfile]: We don't want to add other entrypoints, only ensure that an entrypoint can be specified and is ignored in favour of the default one.

File: images/copilot-bootstrap/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# `copilot-bootstrap` Image
+
+This image is used as a very light placeholder when deploying DBT PaaS infrastructure. It allows us to get the infrastructure into a decent position before we start involving application code and all the variables that adds.
```
Comment (yusufsheiqh) [inline images/copilot-bootstrap/README.md:3]:
```suggestion
This image is used as a very light placeholder when deploying DBT Platform infrastructure. It allows us to get the infrastructure into a decent position before we start involving application code and all the variables that adds.
```

File: images/copilot-bootstrap/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+As it will almost never change, there is no automated build and publish at present, these are manual steps.
+
+The image is published to `public.ecr.aws/uktrade/copilot-bootstrap:latest` in the Tools AWS account.
```
Comment (yusufsheiqh) [inline images/copilot-bootstrap/README.md:7]:
```suggestion
The image is published to `public.ecr.aws/uktrade/copilot-bootstrap:latest` in the `Tools` AWS account.
```

File: images/copilot-bootstrap/default.conf.template [hunk 4]
```diff
@@ -0,0 +1,21 @@
+server {
+    #listen       ${PORT};
```
Comment (yusufsheiqh) [inline images/copilot-bootstrap/default.conf.template:2]: Do we need this?

```suggestion
```

File: images/copilot-bootstrap/index.html [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+<title>Welcome to nginx!</title>
+<style>
+html { color-scheme: light dark; }
+body { width: 35em; margin: 0 auto;
+font-family: Tahoma, Verdana, Arial, sans-serif; }
+</style>
+</head>
+<body>
+<h1>Bootstrap complete!</h1>
+<p>Now swap the manifest.yml image_location with your service image and redeploy.</p>
```
Comment (yusufsheiqh) [inline images/copilot-bootstrap/index.html:13]:
```suggestion
<p>Update the image_location property in the manifest.yml file with your service image and redeploy.</p>
```

Comment (WillGibson) [reply images/copilot-bootstrap/index.html:13]:
```suggestion
<p>Update the image_location property in your service's manifest.yml file with your service image and redeploy.</p>
```

**Review by yusufsheiqh — APPROVED** [review]

File: images/copilot-bootstrap/index.html [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+<title>Welcome to nginx!</title>
+<style>
+html { color-scheme: light dark; }
+body { width: 35em; margin: 0 auto;
+font-family: Tahoma, Verdana, Arial, sans-serif; }
+</style>
+</head>
+<body>
+<h1>Bootstrap complete!</h1>
+<p>Update the image_location property in your service's manifest.yml file with your service image and redeploy.</p>
```
Comment (acodeninja) [inline images/copilot-bootstrap/index.html:13]:
```suggestion
<p>Update the <code>image.location</code> property in your service's <code>manifest.yml</code> file with your service image and redeploy.</p>
```

**Review by acodeninja — APPROVED** [review]
Small change, otherwise great!

**Pull request merged**